ui/pages/login/index.js

```diff
@@ -138,7 +140,20 @@ export default function Login() {
       )
     } catch (e) {
       console.error(e)
-      setError('Invalid credentials')
+      if (e.fieldErrors) {
+        const errors = {}
+        for (const error of e.fieldErrors) {
+          errors[error.fieldName.toLowerCase()] =
+            error.errors[0] || 'invalid value'
+        }
+        setErrors(errors)
+      } else {
+        if (e.code === 401 && e.message === 'unauthorized') {
+          setError('Invalid Credentials')
```

Comment:
We should keep consistent casing here with other errors. I think only the first letter should be capitalized?